nextcord/message.py

```diff
@@ -53,6 +53,7 @@
 from .file import File
 from .flags import MessageFlags
 from .guild import Guild
+from .interactions import MessageInteraction
```

Comment:
This is causing a circular import since `interactions.py` imports from `message.py` already.

There are few ways to solve this that I can think of:
1. Move `MessageInteraction` into `message.py`
2. Make a separate file for the `MessageInteraction` class to import it from
3. Create a helper method on ConnectionState to create message interaction objects (not ideal)
4. Move `from .interactions import MessageInteraction` into the Message `__init__` (hacky solution, not ideal)

Possibly someone can come up with a better idea, but something has to be done to make this run at all.